src/components/Toaster/index.tsx

```diff
@@ -7,31 +7,36 @@ import './toaster.scss'
 export const Toast: any = (
   type: ToastProps['type'],
   title: ToastProps['title'],
+  position: ToastProps['position'],
```

Comment:
Once this becomes optional again, let's make it come after message since it will be more likely that we will want to use a message and the default position.